test/unit/dk_tests.rb

```diff
@@ -0,0 +1,40 @@
+require 'assert'
+require 'dk'
+
+require 'dk/config'
+
+module Dk
+
+  class UnitTests < Assert::Context
+    desc "Dk"
+    setup do
+      @dk_module = Dk
+    end
+    subject{ @dk_module }
+
+    should have_imeths :config, :configure, :init
+
+    should "know its config" do
+      assert_instance_of Config, subject.config
+    end
+
+    should "add init procs to its config on `configure`" do
+      assert_equal [], subject.config.init_procs
+
+      @init_block = proc{}
```

Comment:
@kellyredding - This is really minor, but it doesn't have to be an ivar. I don't care much either way, so if you want to keep it an ivar, cool by me.